From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Li Fei1 <fei1.li@intel.com>
Date: Tue, 25 Feb 2020 11:20:07 +0800
Subject: [PATCH] config: enable NVME for UOS

Build-in NVME driver for UOS by default.

Tracked-On: projectacrn/acrn-hypervisor#4427
Signed-off-by: Li Fei1 <fei1.li@intel.com>
---
 kernel_config_uos | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/kernel_config_uos b/kernel_config_uos
index f5f7586d25f6..5c2a0ede4528 100644
--- a/kernel_config_uos
+++ b/kernel_config_uos
@@ -1552,7 +1552,9 @@ CONFIG_VIRTIO_BLK=y
 #
 # NVME Support
 #
-# CONFIG_BLK_DEV_NVME is not set
+CONFIG_NVME_CORE=y
+CONFIG_BLK_DEV_NVME=y
+# CONFIG_NVME_MULTIPATH is not set
 # CONFIG_NVME_FC is not set
 # CONFIG_NVME_TARGET is not set
 
-- 
https://clearlinux.org

